src/primitives/block.cpp

```diff
@@ -10,21 +10,74 @@
 #include <utilstrencodings.h>
 #include <crypto/common.h>
 #include <streams.h>
+#include <crypto/ethash/helpers.hpp>
+#include <crypto/ethash/include/ethash/progpow.hpp>
+#include <pow.h>
+#include <crypto/randomx/randomx.h>
```

Comment:
All the above added `include` should be removed from here and added to `hash.h / hash.cpp` with all hash functions kept there. There should only be serialization in this file.

Reply:
Done